site/src/components/HelpTooltip/HelpTooltip.tsx

```diff
@@ -1,170 +1,109 @@
-    throw new Error(
-      "This hook should be used in side of the HelpTooltipContext.",
-    );
-  }
-
-  return helpTooltipContext;
+export const HelpTooltip: FC<ComponentProps<typeof Popover>> = (props) => {
+  return <Popover mode="hover" {...props} />;
 };
 
-interface HelpPopoverProps extends PopoverProps {
-  onOpen: () => void;
-  onClose: () => void;
-}
-
-export const HelpPopover: FC<HelpPopoverProps> = ({
-  onOpen,
-  onClose,
-  children,
-  ...props
-}) => {
-  const popover = useClassName(classNames.popover, []);
-  const paper = useClassName(classNames.paper, []);
+export const HelpTooltipContent = (
+  props: ComponentProps<typeof PopoverContent>,
+) => {
+  const theme = useTheme();
 
   return (
-    <Popover
-      className={popover}
-      classes={{ paper }}
-      onClose={onClose}
-      anchorOrigin={{
-        vertical: "bottom",
-        horizontal: "left",
-      }}
-      transformOrigin={{
-        vertical: "top",
-        horizontal: "left",
-      }}
-      PaperProps={{
-        onMouseEnter: onOpen,
-        onMouseLeave: onClose,
-      }}
+    <PopoverContent
       {...props}
-    >
-      {children}
-    </Popover>
+      css={{
+        "& .MuiPaper-root": {
+          fontSize: 14,
+          width: 304,
+          padding: 20,
+          color: theme.palette.text.secondary,
+        },
+      }}
+    />
   );
 };
 
-export interface HelpTooltipProps {
-  // Useful to test on storybook
-  open?: boolean;
+type HelpTooltipTriggerProps = HTMLAttributes<HTMLButtonElement> & {
   size?: Size;
-  icon?: Icon;
-  buttonStyles?: Interpolation<Theme>;
-  iconStyles?: Interpolation<Theme>;
-  children?: ReactNode;
-}
+  hoverEffect?: boolean;
+};
 
-export const HelpTooltip: FC<HelpTooltipProps> = ({
-  children,
-  open = false,
-  size = "medium",
-  icon: Icon = HelpIcon,
-  buttonStyles,
-  iconStyles,
-}) => {
+export const HelpTooltipTrigger = forwardRef<
+  HTMLButtonElement,
+  HelpTooltipTriggerProps
+>((props, ref) => {
   const theme = useTheme();
-  const anchorRef = useRef<HTMLButtonElement>(null);
-  const [isOpen, setIsOpen] = useState(open);
-  const id = isOpen ? "help-popover" : undefined;
-
-  const onClose = () => {
-    setIsOpen(false);
-  };
+  const {
+    size = "medium",
+    children = <HelpTooltipIcon />,
+    hoverEffect = true,
+    ...buttonProps
+  } = props;
+
+  const hoverEffectStyles = css({
+    opacity: 0.5,
+    "&:hover": {
+      opacity: 0.75,
+    },
+  });
 
   return (
-    <>
+    <PopoverTrigger>
       <button
-        ref={anchorRef}
-        aria-describedby={id}
+        {...buttonProps}
+        aria-label="More info"
+        ref={ref}
         css={[
           css`
             display: flex;
             align-items: center;
             justify-content: center;
-            width: ${theme.spacing(getButtonSpacingFromSize(size))};
-            height: ${theme.spacing(getButtonSpacingFromSize(size))};
-            padding: 0;
+            padding: 4px 0;
```

Comment:
I'm skeptic about forcing this padding here, and not at a container level. this'll be really hard to undo when it looks wrong.

Reply:
Removed!